extensions/AbuseFilter/Views/AbuseFilterViewDiff.php

```diff
@@ -135,7 +135,7 @@ function loadFromHistoryRow( $row ) {
 			'meta' => array(
 				'history_id' => $row->afh_id,
 				'modified_by' => $row->afh_user,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "modified_by" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)